fixtures/eurolite/led-bar-6-qcl-rgbw.json

```
@@ -14,9 +14,11 @@
   },
   "links": {
     "manual": [
+      "https://www.steinigke.de/download/51930399-Manual-120629-1.100-eurolite-led-bar-6-qcl-rgbw-bar-de_en.pdf",
       "https://images.static-thomann.de/pics/atg/atgdata/document/manual/420780_eurolite_led_bar_6_12_rgbw_de_uk.pdf"
     ],
     "productPage": [
+      "https://www.steinigke.de/en/mpn51930399-eurolite-led-bar-6-qcl-rgbw-bar.html",
       "https://www.thomann.de/gb/eurolite_led_bar_6_qcl_rgbw.htm"
     ],
```

I think only the official links on steinigke.de are sufficient, no need to link to a distributor's website additionally if the manual is identical.

```suggestion
  "links": {
    "manual": [
      "https://www.steinigke.de/download/51930399-Manual-120629-1.100-eurolite-led-bar-6-qcl-rgbw-bar-de_en.pdf"
    ],
    "productPage": [
      "https://www.steinigke.de/en/mpn51930399-eurolite-led-bar-6-qcl-rgbw-bar.html"
    ],
```

---

I didn't want to delete existing links, so I just added new ones.
Now those old ones are gone.